pkg/push/api/api.go

```diff
@@ -720,17 +864,35 @@ func (s *PushService) DeletePush(
 		if err != nil {
 			return err
 		}
-		handler, err := command.NewPushCommandHandler(editor, push, s.publisher, req.EnvironmentNamespace)
+		prev := &domain.Push{}
+		if err = copier.Copy(prev, push); err != nil {
+			return err
+		}
+		push.Deleted = true
+		event, err = domainevent.NewEvent(
+			editor,
+			eventproto.Event_PUSH,
+			push.Id,
+			eventproto.Event_PUSH_CREATED,
+			&eventproto.PushCreatedEvent{
+				FcmServiceAccount: push.FcmServiceAccount,
+				Tags:              push.Tags,
+				Name:              push.Name,
+			},
+			req.EnvironmentNamespace,
+			push,
+			prev,
+		)
 		if err != nil {
 			return err
 		}
-		if err := handler.Handle(ctx, req.Command); err != nil {
+		if err = s.publisher.Publish(ctx, event); err != nil {
 			return err
 		}
 		return pushStorage.UpdatePush(ctx, push, req.EnvironmentNamespace)
 	})
 	if err != nil {
-		if err == v2ps.ErrPushNotFound || err == v2ps.ErrPushUnexpectedAffectedRows {
+		if errors.Is(err, v2ps.ErrPushNotFound) || errors.Is(err, v2ps.ErrPushUnexpectedAffectedRows) {
```

Comment:
Just confirming.
Are you going to fix to return success when it gets an `ErrPushUnexpectedAffectedRows` error in this PR?

Reply:
Yes I missed that, updated